app/main.cpp

```diff
@@ -1,9 +1,38 @@
 #include "app.h"
+#include <iostream>
+
+using namespace vsite::oop::v3;
 
 // read number of results
 // read all student data
 // read grade and output grade statistics
 // read letter and output letter statistics
 int main()
 {
+
+	std::cout << "Enter number of students: ";
+	unsigned n;
+	std::cin >> n;
+	results r(n);
+
+	std::string name;
+	unsigned grade;
+	unsigned i(0);
+	while (i < n) {
+
+		std::cout << "Enter " << i + 1 << ". student's name : ";
+		std::cin >> name;
+		std::cout << "Enter student grade: ";
+		std::cin >> grade;
+		r.add({ name, static_cast<int>(grade) });
```

Comment:
Zašto ovdje `cast`? Ili napravite `grade` `unsigned int` ili ovdje učitavajte `int`.